apps/prairielearn/src/cron/sendExternalGraderDeadLetters.js

```diff
@@ -38,13 +38,11 @@ module.exports.run = (callback) => {
         msg = jobsMessages + resultsMessages;
       },
       async () => {
-        const res = await opsbot.sendMessage(msg);
-        if (res && res.status !== 200) {
-          logger.error(
-            `Error posting external grading dead letters to slack [status code ${res.status}]`,
-            await res.text(),
+        await opsbot
+          .sendMessage(msg)
+          .catch(async (err) =>
```

Comment:
Nit: should be consistent in using (or not) the `async` keyword with `.catch(...)` functions. I'd err towards not, as long as the function doesn't actually have to be async.

Reply:
Removed in 80c7bb2.